services/airtime/dtone/client.go

```diff
@@ -164,7 +170,7 @@ func (c *Client) TransactionSync(externalID string, productID int, mobileNumber
 		},
 	}
 
-	trace, err := c.request("POST", "sync/transactions", payload, &response)
+	trace, err := c.request("POST", "async/transactions", payload, &response)
```

Comment:
synchronous approach was deprecated https://dvs-api-doc.dtone.com/#tag/Transactions

Reply:
I can see that but isn't this a problem? Doesn't this require that we poll it to know if it actually succeeded?

Reply:
I think we get 201 HTTP status for each transaction we send to DT One and it will send that in an async task

we could add the callback to actually update the transfer in case that is not successful

I understand that slight changes what success mean in the flow but most user will prefer having that instead of not being able to use that at all I guess.
so in the flow success mean we send the requests to DT One and we hade a transaction ID back successfully

Reply:
Since the sync version is only deprecated, can't we just keep using it for now until we figure out how a flow can do async transfers? 

I think to do it correctly, transfer_airtime actions would have to become waits, and then have mailroom listen for success or failure, and resume the flow session accordingly.